ustriage/task.py

```diff
@@ -145,6 +145,39 @@ def short_title(self):
         }[self.obj.target.resource_type_link]
         return ' '.join(self.title.split(' ')[start_field:]).replace('"', '')
 
+    def get_releases(self, open_bug_statuses):
+        """List of chars reflecting per release status.
+
+        Gets a list of chars, one per supported release that show if that task
+        exists (present) and is open (lower case) or closed (upper case).
+
+        Note: This has to stay a fixed length string to maintain the layout
+        """
+        releaseinfo = ''
+
+        # breaking the URL is faster than checking it all through API
+        for task in self.obj.bug.bug_tasks:
+            task_elements = str(task).split('/')
+            # skip root element and other projects
+            if task_elements[4] != 'ubuntu':
+                continue
+            # Only care for the task that we high-level report about
+            if task_elements[-3] != str(self.src):
+                continue
+
+            # get first char of release (devel = d)
+            relchar = task_elements[5][0]
```

Comment:
Still in the readability path, could we call this sth like release_char?

Reply:
sure, done as well